00_introduction/21_architecture_nodes.txt

```diff
@@ -33,18 +33,27 @@ available for these platforms:
 
 * Ubuntu 12.10 (quantal)
 
+* IBM AIX 5.3
+
+* IBM AIX 6.1
+
+* IBM AIX 7.1
+
 [TIP]
 
 [[install-on-windows, Install on Microsoft Windows]] 
 
-.Windows Nodes 
+.Windows and AIX Nodes
 
 ====
 
-Installing Rudder on Windows requires the commercial version of CFEngine (named Nova).
+* On Windows, installing Rudder requires the commercial version of CFEngine (named Nova)
+* For IBM AIX, the agent is open-source but the pre-built RPM packages are distributed
+  by Normation under commercial license.
```

Comment:
Hmmm. Almost :)

Actually we don't change the license when distributing (just like Red Hat "sells" DVDs of open source software). Could you rephrase to clarify as in "For IBM AIX, pre-built RPM packages are distributed by Normation only."

And while you're here, can you s/Nova/Enterprise/ ? :)